condition/scenario_conditions/CMakeLists.txt

```diff
@@ -1,51 +1,28 @@
-cmake_minimum_required(VERSION 2.8.3)
+cmake_minimum_required(VERSION 3.5)
 project(scenario_conditions)
 
-## Compile as C++11, supported in ROS Kinetic and newer
-add_compile_options(-std=c++14)
+### Compile options
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+  set(CMAKE_CXX_STANDARD_REQUIRED ON)
+  set(CMAKE_CXX_EXTENSIONS OFF)
+endif()
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wno-unused-parameter -Wall -Wextra -Wpedantic)
+endif()
 
-find_package(catkin REQUIRED COMPONENTS
-  pluginlib
-  roscpp
-  scenario_api
-  scenario_intersection
-  scenario_utility
-  visualization_msgs
-)
-
-catkin_package(
-  INCLUDE_DIRS include
-  LIBRARIES scenario_conditions
-  CATKIN_DEPENDS
-    pluginlib
-    roscpp
-    scenario_api
-    scenario_intersection
-    scenario_utility
-    visualization_msgs
-)
+### Dependencies
+find_package(ament_cmake_auto REQUIRED)
+ament_auto_find_build_dependencies()
 
-include_directories(
-  include
-  ${catkin_INCLUDE_DIRS}
-  ${YAML_CPP_INCLUDE_DIR}
+set(SCENARIO_CONDITIONS_SRC
```

Comment:
[nit] I would omit the variable and just add the file verbatim to the library below

Reply:
Done.